.travis.yml

```diff
@@ -0,0 +1,12 @@
+language:
+  - node_js
+node_js:
+  - 0.10.25
+
+before_install:
+  - npm install -g gulp
```

Comment:
Pro tip: Because you've defined `gulp` as a dependency in package.json it will be pulled into the `node_modules`. Dependencies with binaries (such as `gulp`) get symlinked into `node_modules/.bin/`. This means that in the `script:` line you could run: `./node_modules/.bin/gulp default` and avoid installing gulp globally.